package.json

```diff
@@ -44,4 +44,4 @@
     "yargs": "^14.0.0",
     "yargs-unparser": "^1.5.0"
   }
-}
+}
```

Comment:
Can you include the end of file line here? There's a `.editorconfig` file in the src that contains the rules. If you could also fixup these commits to be a single change too that would be great. Thank you.